interface-impl-v2/src/main/java/org/mule/apikit/implv2/v10/model/ParameterImpl.java

```diff
@@ -228,4 +237,13 @@ private boolean needsQuotes(TypeDeclaration typeDeclaration) {
     return !(isOfType(type, NUMBER_DATA_TYPES) || isOfType(type, BOOLEAN_DATA_TYPES));
   }
 
+  private static boolean hasNilShape(UnionTypeDeclaration unionType) {
+    for (TypeDeclaration type : unionType.of()) {
+      if (type instanceof NullTypeDeclaration) {
+        return true;
+      }
+    }
+    return false;
+  }
+
```

Comment:
You can update this implementation as you did with the suggested by @iglosiggio before